src/ansys/mechanical/core/embedding/enum_importer.py

```diff
@@ -0,0 +1,35 @@
+# Copyright (C) 2023 ANSYS, Inc. and/or its affiliates.
+# SPDX-License-Identifier: MIT
+#
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+"""Library to import Mechanical enums.
+
+A useful subset of what is imported by
+Ansys Inc/v{NNN}/ACT/apis/Mechanical.py
+"""
+
+import clr
+
```

Comment:
@klmcadams I'm getting errors in the CI like this:
src/ansys/mechanical/core/embedding/enum_importer.py:33:1: F401 'Ansys' imported but unused
src/ansys/mechanical/core/embedding/enum_importer.py:34:1: F403 'from Ansys.ACT.Interfaces.Common import *' used; unable to detect undefined names
src/ansys/mechanical/core/embedding/enum_importer.py:34:1: F401 'Ansys.ACT.Interfaces.Common.*' imported but unused
src/ansys/mechanical/core/embedding/enum_importer.py:35:1: F403 'from Ansys.Mechanical.DataModel.Enums import *' used; unable to detect undefined names
src/ansys/mechanical/core/embedding/enum_importer.py:35:1: F401 'Ansys.Mechanical.DataModel.Enums.*' imported but unused
3     F401 'Ansys' imported but unused
2     F403 'from Ansys.ACT.Interfaces.Common import *' used; unable to detect undefined names
5

How can I ignore these? These are indeed used